### DISCUSSION TITLE
latest version 0.0.3 is not available in mvn repository?

### DISCUSSION THREAD

**Discussion by 4k5h4y** [Auties00/WhatsappWeb4j#451] [General]
@Auties00  I saw you have addded new six parts login feature in master branch but i see that the latest 0.0.3 jar is not available in mvn repository. can you help with this please? thanks!!

**Comment by codespearhead** [comment]
@4k5h4y Can you confirm whether the problem is solved? v 0.0.3 seems to have been published on [February 25, 2024](https://mvnrepository.com/artifact/com.github.auties00/cobalt).
